tests/ses/nodes_preparation.pm

```diff
@@ -16,38 +16,33 @@ use strict;
 use testapi;
 use mm_network;
 use lockapi;
-use utils 'systemctl';
+use utils qw(systemctl zypper_call);
+use version_utils 'is_sle';
+
+sub restart_and_sync_chrony {
+    systemctl "restart chronyd";
+    systemctl "status chronyd";
+    assert_script_run 'chronyc waitsync 40 0.01 && chronyc sources', 400;
+}
 
 sub run {
     select_console 'root-console';
-    # configure and start ntp, ntp server for nodes is master
     if (check_var('HOSTNAME', 'master')) {
-        my $all_ses_nodes = get_var('NODE_COUNT') + 1;
         # create mutex lock and barriers
         mutex_create('master_ready');
-        barrier_create('network_configured', $all_ses_nodes);
-        barrier_create('deployment_done',    $all_ses_nodes);
-        barrier_create('all_tests_done',     $all_ses_nodes);
-        assert_script_run 'echo \'server ntp1.suse.de burst iburst\' >> /etc/ntp.conf';
+        my $all_ses_nodes = get_var('NODE_COUNT') + 1;
+        barrier_create('network_configured',  $all_ses_nodes);
+        barrier_create('master_chrony_ready', $all_ses_nodes);
+        barrier_create('deployment_done',     $all_ses_nodes);
+        barrier_create('all_tests_done',      $all_ses_nodes);
     }
     else {
-        assert_script_run 'echo \'server master.openqa.test burst iburst\' >> /etc/ntp.conf';
         mutex_lock('master_ready');
         mutex_unlock('master_ready');
     }
-    # print the ntp config
-    assert_script_run 'grep -v ^# /etc/ntp.conf';
-    systemctl 'restart ntpd';
     # disable ipv6
     assert_script_run 'echo \'net.ipv6.conf.all.disable_ipv6 = 1\' >> /etc/sysctl.conf';
```

Comment:
No need to restart network on "DEEPSEA_TESTSUITE" nodes after `net.ipv6.conf.all.disable_ipv6=1` ?